docs-content/general/6_product-features/1_session-replay/1_overview.md

```diff
@@ -42,7 +42,7 @@ Read more about the features we support in Session Replay below:
     <DocsCard title="Tracking Users & Events."  href="./events-and-users.md">
         {"SDK support for tracking users and their corresponding actions throughout a session."}
     </DocsCard>
-    <DocsCard title="GrqphQl"  href="./graphql.md">
+    <DocsCard title="GraphQl"  href="./graphql.md">
```

Comment:
Did you want to capitalize the L in QL as well?

Reply:
Yep! Thanks for catching that.